pkg/cmd/secret/set/set.go

```diff
@@ -121,6 +122,7 @@ func NewCmdSet(f *cmdutil.Factory, runF func(*SetOptions) error) *cobra.Command
 	}
 	cmd.Flags().StringVarP(&opts.OrgName, "org", "o", "", "Set a secret for an organization")
 	cmd.Flags().StringVarP(&opts.EnvName, "env", "e", "", "Set a secret for an environment")
+	cmd.Flags().BoolVarP(&opts.UserSecrets, "user", "u", false, "Set a secret for the current user")
```

Comment:
Are these secrets used for anything else other than Codespaces? Can we call them "codespace secrets" instead of "user secrets"? To me, "user secret" sounds vague and doesn't evoke connection to Codespaces at all.

Reply:
Secrets are attached to an entity (user, org, repo, environment) and associated with some feature area/product (Actions, Codespaces, Dependabot).

Both Codespaces and Dependabot support secrets at the repository and organization level - 
https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/managing-encrypted-secrets-for-your-repository-and-organization-for-codespaces
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/managing-encrypted-secrets-for-dependabot


Currently, the only non-Actions secrets with a API support are Codespaces user secrets.

What do you think about adding a new parameter to choose between Actions/Codespaces?
i.e.
`bin/gh secret set -u -codespaces`

or 

`bin/gh secret set -u --app=Codespaces`

We could add some validation to prevent someone from specifying Codespaces or Actions for an unsupported entity type

Reply:
Let's hold off on the new parameter and keep just `--user` for the sake of simplicity. Our added docs should clarify what "user secrets" are useful for at this moment. 👍